framework/core/js/src/common/Application.tsx

```diff
@@ -262,7 +263,22 @@ export default class Application {
   }
 
   public boot() {
-    this.initializers.toArray().forEach((initializer) => initializer(this));
+    const caughtInitializationErrors: CallableFunction[] = [];
+
+    this.initializers.toArray().forEach((initializer) => {
+      try {
+        initializer(this);
+      } catch (e) {
+        const extension = initializer.itemName.includes('/')
+          ? initializer.itemName.replace(/(\/flarum-ext-)|(\/flarum-)/g, '-')
+          : initializer.itemName;
+
+        caughtInitializationErrors.push(() => fireApplicationError({
+          userTitle: extractText(app.translator.trans('core.lib.error.extension_initialiation_failed_message', { extension })),
+          consoleTitle: `${extension} failed to initialize`,
```

Comment:
Why keep this untranslated?

Reply:
I agree with that there's no reason to translate this.